meta-genivi-dev/recipes-sota/rvi-sota-client/rvi-sota-client_git.bb

```diff
@@ -112,10 +112,9 @@ SRC_URI[index.sha256sum] = "86114b93f1f51aaf0aec3af0751d214b351f4ff9839ba031315c
 
 SYSTEMD_SERVICE_${PN} = "sota_client.service"
 
-DEPENDS += " openssl "
+DEPENDS += " openssl dbus "
```

Comment:
I did a bit of digging and seems dbus as build-dep was added in 2015: https://github.com/advancedtelematic/rvi_sota_client/blob/master/Cargo.toml#L21

So it's most likely that we have been just lucky so far that dbus was built before sota-client. could you please update the commit logs?